editor/plugins/script_text_editor.cpp

```diff
@@ -1598,8 +1598,27 @@ static Node *_find_script_node(Node *p_edited_scene, Node *p_current_node, const
 	return nullptr;
 }
 
+
+String ScriptTextEditor::_escape_drop_data(const String &str) {
+	// This function escapes resource paths, node paths, or object properties
+	// according to String delimiter preferences.
+	
+	String escaped = str.c_escape();
+
+	// Resource paths, node paths and object properties cannot contain double-quotes
+	// so we do not bother reverting back the .replace from c_escape.
```

Comment:
This comment seems a bit out of place.

Reply:
Agreed, leftover from a previous understanding of the situation in which I thought filenames could have double-quotes. They cannot, so now we just don't bother worrying about them :)